packages/core/src/utils/helpers.css

```diff
@@ -55,3 +55,61 @@
   font-size: var(--fontsize-body-s);
   line-height: var(--lineheight-xl);
 }
+
+/* HEADINGS */
+
+.heading-xxl {
+  font-size: var(--fontsize-heading-xxl);
+  font-weight: 400;
+  line-height: var(--lineheight-s);
+  letter-spacing: -1.2px;
+}
+
+.heading-xl {
+  font-size: var(--fontsize-heading-xl);
+  font-weight: 400;
+  line-height: var(--lineheight-s);
+  letter-spacing: -1.0px;
+}
+
+.heading-xl-mobile {
+  font-size: var(--fontsize-heading-xl-mobile);
+  font-weight: 400;
+  line-height: var(--lineheight-s);
+  letter-spacing: -0.8px;
+}
+
+.heading-l {
+  font-size: var(--fontsize-heading-l);
+  font-weight: 400;
+  line-height: var(--lineheight-s);
+  letter-spacing: -0.4px;
+}
+
+.heading-m {
+  font-size: var(--fontsize-heading-m);
+  font-weight: 500;
+  line-height: 1.33;
+  letter-spacing: -0.2px;
+}
+
+.heading-s {
+  font-size: var(--fontsize-heading-s);
+  font-weight: 700;
+  line-height: 1.4;
```

Comment:
We are now partly using line height tokens here. Mostly the `lineheight-s` token is used and other than that we use custom values. I'm wondering what is the point of having the line height tokens if we cannot use them.

Possibly remove line height tokens or add these other values as tokens? Or maybe leave 1 or 2 line height tokens; such as 1.0 and 1.5?